index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="/styles.css" />
+  </head>
+  <body>
+    <div class="wrapper">
+      <header class="header-box">
+        <p>John Doe</p>
+        <img src="/images/Menu-icon.png" alt="" />
+      </header>
+        <section class="name-info">
+          <h1>Ukwishaka Levy</h1>
+          <h2>Software Developer</h2>
+        </section>
+        <section class="box-tools">
+          <p id="intro">
+            I can help you build a product , feature or website Look through
+            some of my work and experience! If you like what you see and have a
+            project you need coded, don’t hesitate to contact me.
+          </p>
+          <img src="/images/vector-line.png" alt="line" />
+          <div class="skills-box">
+            <div>
+              <h4>Languages</h4>
+              <p>Javascript</p>
+              <p>Ruby</p>
+              <p>Html</p>
+              <p>Css</p>
+            </div>
+            <div>
+              <h4>Frameworks</h4>
+              <p>Bootstrap</p>
+              <p>Ruby on Rails</p>
+              <p>RSPec</p>
+              <p>CapyBara</p>
+              <p>Selenium</p>
+            </div>
+            <div>
+              <h4>Skills</h4>
+              <p>Codekit</p>
+              <p>Github</p>
+              <p>Codepen</p>
+              <p>Gitlab</p>
+              <p>Terminal</p>
+            </div>
+          </div>
+          <img src="/images/vector-line.png" alt="line" />
+        </section>
+      </div>  
+        <div class="support_items">
+          <div class="social-media">
+            <img src="/images/Github.png" alt="" />
+            <img src="/images/Icon-Linkedin.png" alt="Linkerdin" />
+            <img src="/images/Vector3.png" alt="Vector3" />
+            <img src="/images/twitter.png" alt="Twitter" />
+            <img src="/images/MVector.png" alt="M-vector" />
+          </div>
```

Comment:
- [ ] Please consider using a `ul` tag to wrap all the social media icons and use  `li` tag to wrap each of the social media icons. This is to improve the semantic structure of your page and improve the code structure too.

- [ ]  Also, each icon should be wrapped with an `a` tag which will link to your social media handles.
See suggestion below:

```suggestion
          <ul class="social-media">
            <li>
               <a href=""><img src="/images/Github.png" alt="Github" /></a>
            <li>
            <li>
               <a href=""><img src="/images/Icon-Linkedin.png" alt="Linkedin" /></a>
             <li>
             <li>
               <a href=""><img src="/images/Vector3.png" alt="Angelist" /></a>
             <li>
             <li>
               <a href=""><img src="/images/twitter.png" alt="Twitter" /></a>
             <li>
             <li>
               <a href=""><img src="/images/MVector.png" alt="Medium" /></a>
             <li>
          </ul>
```